UsbIpServer/RegistryWatcher.cs

```diff
@@ -0,0 +1,61 @@
+﻿// SPDX-FileCopyrightText: Microsoft Corporation
+//
+// SPDX-License-Identifier: GPL-2.0-only
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management;
+using System.Threading;
+
+namespace UsbIpServer
+{
+    sealed public class RegistryWatcher : IDisposable
```

Comment:
I don't like `public` classes in EXE assemblies (unless they really are exported). For internal classes that are not explicitly instantiated (i.e. instantiated only by dependency injection), there is:

` [SuppressMessage("Performance", "CA1812:Avoid uninstantiated internal classes", Justification = "Instantiated by DI")]`

Reply:
fixed